src/kanata/mod.rs

```diff
@@ -529,6 +551,12 @@ impl Kanata {
                 }
             }
         }
+        #[cfg(all(target_os = "windows", feature = "gui"))]
+        if let Some(gui_tx) = GUI_TX.get() {
```

Comment:
Let's put the 5 lines of code into `gui_win.rs` and make the diff here be a function call.

Reply:
moved